meveo-admin-ejbs/src/main/java/org/meveo/service/git/GitClient.java

```diff
@@ -534,6 +556,10 @@ public boolean pull(GitRepository gitRepository, String username, String passwor
 
         } finally {
             keyLock.unlock(gitRepository.getCode());
+            
+			if (moduleScript != null) {
+				moduleScriptService.postPull(moduleScript, module);
```

Comment:
Don't do this in the finally, but at then end of the try / catch block. The post pull should only be executed if the pull has been successfull